src/command.ts

```diff
@@ -217,7 +216,16 @@ export class Command {
               ])
             );
           }
-          process.exit();
+          process.exitCode = 0;
+          const timeout = setTimeout(
+            (cmd: any) => {
+              track(cmd, "terminated_with_open_handlers");
+              process.exit(0);
+            },
+            2000,
```

Comment:
Consider pushing this even higher than 2s - since this is a failsafe, I'm ok with 10s even